src/main/java/Controllers/WelcomeController.java

```diff
@@ -84,20 +84,21 @@ public void handleLogin(ActionEvent actionEvent) {
                     VistaNavigator.setMainController(loader.getController());
                     // Set data in the controller
                     mainController.setUserName(userName);
-                    mainController.setUserRole(response);
+                    JSONObject bla = new JSONObject(response);
```

Comment:
Maybe consider more meaningful name.

Reply:
bla = big long answer
ill change to somthing else